hdf5-derive/src/lib.rs

```diff
@@ -1,19 +1,22 @@
 #![recursion_limit = "192"]
 
 extern crate proc_macro;
```

Comment:
Hm, do we even need these `extern crate` anymore?

Reply:
Let's see if this works with our `MSRV`

Reply:
This works but a CI job is halting (most likely unrelated)

Reply:
Unrelated, I think, otherwise it'd crash immediately